src/30_Advanced/amp-user-notification_with_Server_Endpoint.html

```diff
@@ -21,12 +21,12 @@
     in the left bottom corner.
   -->
   <style amp-custom>
-    footer {
-      margin-bottom: 50px;
-    }
-    amp-user-notification {
-      padding: 8px;
-      background: beige;
+    /* Center the notification content */
+    amp-user-notification > div {
+      padding: 1rem;
```

Comment:
thought about the same - in this case we have all relevant styling in once place. We need some clear rules on when to use basscss and when not.